api-specification.md

```diff
@@ -0,0 +1,652 @@
+                        "name":"contact_id",
+                        "title":"Contact ID",
+                        "type":"string"
+                     },
+                     {  
+                        "name":"question_id",
+                        "title":"Question ID",
+                        "type":"string"
+                     },
+                     {  
+                        "name":"response_id",
+                        "title":"Response ID",
+                        "type":"any"
+                     },
+                     {  
+                        "name":"response_metadata",
+                        "title":"Response Metadata",
+                        "type":"object"
+                     }
+                  ],
+                  "questions":[  
+                     {  
+                        "1448506769745_42":{  
+                           "type":"select_one",
+                           "label":"Are you a woman or a man?",
+                           "type_options":{  
+                              "choices":[  
+                                 "Woman",
+                                 "Man",
+                                 "Other"
+                              ]
+                           }
+                        }
+                     },
+                     {  
+                        "1448506773018_89":{  
+                           "type":"numeric",
+                           "label":"How old are you? Please enter your age in years.",
+                           "type_options":{  
+                              "range":[  
+                                 -99,
+                                 99
+                              ]
+                           }
+                        }
+                     },
+                     {  
+                        "1448506774930_30":{  
+                           "type":"open",
+                           "label":"What was the best thing that happened to you today?",
+                           "type_options":{  
+
+                           }
+                        }
+                     }
+                  ]
+               }
+            }
+         ]
+      }
+   }
+}
+```
+
+_Response_:
+
+The response from the server must adhere to the [JSON API specification](http://jsonapi.org/format/#crud-creating-responses) for POST responses.
+
+_Response example:_
+
+```
+HTTP/1.1 201 Created
+Location: https://go.votomobile.org/flow-results/packages/0c364ee1-0305-42ad-9fc9-2ec5a80c55fa
+Content-Type: application/vnd.api+json
+
+{
+  "data": {
+    "type": "packages",
+    "id": "0c364ee1-0305-42ad-9fc9-2ec5a80c55fa",
+    "attributes": {  
+         "profile":"flow-results-package",
+         "name":"standard_test_survey",
+         "flow-results-specification":"1.0.0-rc1",
+         "created":"2015-11-26 02:59:24+00:00",
+         "modified":"2017-12-04 15:54:44+00:00",
+         "id":"0c364ee1-0305-42ad-9fc9-2ec5a80c55fa",
```

Comment:
Do we need the `id` within the `attributes`?

Reply:
Yes, because we are putting the standard (Flow Results descriptor)[https://floip.gitbooks.io/flow-results-specification/content/specification.html#descriptor]  inside the attributes.